src/components/Main.vue

```diff
@@ -8,7 +8,7 @@
     <body>
       <div class="row">
         <div class="leftcolumn">
-          <div class="text-center col-md-4 col-md-offset-4">
+          <div class="text-center input-column col-md-4 col-md-offset-4">
```

Comment:
General question: What's the significance of the `col-md-4` and `col-md-offset-4` classes?

Reply:
Those classes change the size of columns and the space between them.  Right now that is handled by width and margin properties in the CSS at the bottom so these classes shouldn't be necessary.  Nothing appears to change when I remove these classes.